extensions/amp-fx-collection/0.1/providers/fx-provider.js

```diff
@@ -249,6 +279,13 @@ export class FxElement {
     return this.mutateScheduled_;
   }
 
+  /**
+   * @returns {boolean}
```

Comment:
I know what you mean by "repeat" here, but it's very ambiguous out of context. Perhaps add a comment to the jsdoc?

Reply:
Done.